modules/unitbar.lua

```diff
@@ -62,22 +62,6 @@ function GladiusEx:NewUnitBarModule(name, defaults_arena, defaults_party)
 
 			self.frame[unit].statusbar:SetStatusBarColor(color.r, color.g, color.b, color.a or 1)
 			self.frame[unit].icon:SetTexture([[Interface\Glues\CharacterCreate\UI-CharacterCreate-Classes]])
-
-			-- V: wtf is "CLASS_BUTTONS"?
-			if not CLASS_BUTTONS then

```

Comment:
well, the code has been `return`ing early for all my testing, and I never had an issue with class icon not showing up.